test/Geosuggest_spec.tsx

```diff
@@ -642,6 +642,25 @@ describe('Component: Geosuggest', () => {
 
       expect(label).to.not.be.null;
     });
+
+    it('should render a <label> pointing to the input if `label` and `id` props were supplied', () => {
+      const props = {
+        id: 'geosuggest-id',
+        label: 'some label'
+      };
+
+      renderGeosuggest(props);
+
+      const label = component.container.querySelector(
+        'label'
+      ) as HTMLLabelElement;
+
+      const input = component.container.querySelector(
+        'input'
+      ) as HTMLInputElement;
+
+      expect(label).to.be.equal(input);
```

Comment:
This compares the label element with the input element, which will never be the same. To check the connection between those two elements, please compare the appropriate attributes.

```suggestion
      expect(label.getAttribute('for')).to.be.equal(input.getAttribute('id'))
```